rustworkx-core/src/max_weight_matching.rs

```diff
@@ -371,7 +371,7 @@ fn expand_blossom<E>(
         // base.
         assert!(label_ends[blossom].is_some());
         let entry_child = in_blossoms[endpoints[label_ends[blossom].unwrap() ^ 1]];
-        // Decied in which direction we will go around the blossom.
+        // Decide in which direction we will go around the blossom.
```

Comment:
This is an example of a find with low-value,, I don't want to block people when they make a typo on a code comment. Except if that code comment is a docstring of course